### ISSUE TITLE
One way...

### ISSUE THREAD

**Issue by jmr2704** [coreyphillips/bitimage#7]
![image](https://github.com/coreyphillips/bitimage/assets/14109821/ab3c0b14-1529-49d0-aecf-3cc96524b782)

**Comment by jmr2704** [comment]
![image](https://github.com/coreyphillips/bitimage/assets/14109821/ca9440d6-cfec-49b8-bf2a-01d5196d64ae)
Cada dia mais perto <3

**Comment by SmartArt09** [comment]
Wow bro, this is amazing
How are you trying this if I may ask?

**Comment by jmr2704** [comment]
> Wow bro, this is amazing How are you trying this if I may ask?

 I'm doing sentence tests by changing some words from the original sentence

**Comment by SmartArt09** [comment]
> > Wow bro, this is amazing How are you trying this if I may ask?
> 
> I'm doing sentence tests by changing some words from the original sentence



> > Wow bro, this is amazing How are you trying this if I may ask?
> 
> I'm doing sentence tests by changing some words from the original sentence

I noticed you kept a lot of words from the original mnemonic, then changed 2 words from between and the last few words are the starting ones from the original bip39 wordlist. So you are just trying the combinations?
Also, as per his challenge, there only seems to be an optional passphrase, not a different mnemonic. Or are you trying this mnemonic as a passphrase?

**Comment by jmr2704** [comment]
> > > Wow bro, this is amazing How are you trying this if I may ask?
> > 
> > 
> > I'm doing sentence tests by changing some words from the original sentence
> 
> > > Wow bro, this is amazing How are you trying this if I may ask?
> > 
> > 
> > I'm doing sentence tests by changing some words from the original sentence
> 
> I noticed you kept a lot of words from the original mnemonic, then changed 2 words from between and the last few words are the starting ones from the original bip39 wordlist. So you are just trying the combinations? Also, as per his challenge, there only seems to be an optional passphrase, not a different mnemonic. Or are you trying this mnemonic as a passphrase?

I have already tested adding a word or changing just 1, and without success, it even completely deviates from the original wallet.

**Comment by SmartArt09** [comment]
> > > > Wow bro, this is amazing How are you trying this if I may ask?
> > > 
> > > 
> > > I'm doing sentence tests by changing some words from the original sentence
> > 
> > 
> > > > Wow bro, this is amazing How are you trying this if I may ask?
> > > 
> > > 
> > > I'm doing sentence tests by changing some words from the original sentence
> > 
> > 
> > I noticed you kept a lot of words from the original mnemonic, then changed 2 words from between and the last few words are the starting ones from the original bip39 wordlist. So you are just trying the combinations? Also, as per his challenge, there only seems to be an optional passphrase, not a different mnemonic. Or are you trying this mnemonic as a passphrase?
> 
> I have already tested adding a word or changing just 1, and without success, it even completely deviates from the original wallet.

Yeah, from what I know, that's not how wallets are generated. And as mentioned in the challenge, the address with the unclaimed prize uses the same mnemonic as for the kitten.jpg image, just with the optional BIP0039 passphrase, which only Corey knows (hopefully) and is unwilling to share a hint about as of now.

**Comment by jmr2704** [comment]
> > > > > Wow bro, this is amazing How are you trying this if I may ask?
> > > > 
> > > > 
> > > > I'm doing sentence tests by changing some words from the original sentence
> > > 
> > > 
> > > > > Wow bro, this is amazing How are you trying this if I may ask?
> > > > 
> > > > 
> > > > I'm doing sentence tests by changing some words from the original sentence
> > > 
> > > 
> > > I noticed you kept a lot of words from the original mnemonic, then changed 2 words from between and the last few words are the starting ones from the original bip39 wordlist. So you are just trying the combinations? Also, as per his challenge, there only seems to be an optional passphrase, not a different mnemonic. Or are you trying this mnemonic as a passphrase?
> > 
> > 
> > I have already tested adding a word or changing just 1, and without success, it even completely deviates from the original wallet.
> 
> Yeah, from what I know, that's not how wallets are generated. And as mentioned in the challenge, the address with the unclaimed prize uses the same mnemonic as for the kitten.jpg image, just with the optional BIP0039 passphrase, which only Corey knows (hopefully) and is unwilling to share a hint about as of now.

 The problem is that there is no clue as to how many words the password he entered is. Whether it is one or several. Oh, it gets really complicated.

**Comment by lucasdmt** [comment]
testei 400k palavras de uma wordlist, todas as palavras da pagina e do seu perfil X etc... então realmente acho que seja uma frase ou algum HEX assim como no seu bitcoin audio puzzle era.

**Comment by lucasdmt** [comment]
Corey Phillips he said on his twitter "It's not meant to be solved. It's meant to prove the viability of the method" I tested a few thousand combinations using brute force, your method is really safe

**Comment by SmartArt09** [comment]
> Corey Phillips he said on his twitter "It's not meant to be solved. It's meant to prove the viability of the method" I tested a few thousand combinations using brute force, your method is really safe

May I have this tweet link, please? I didn't know he tweeted about it nor did I find anything when I went through his twitter posts.

**Comment by lucasdmt** [comment]
> > Corey Phillips disse em seu twitter "Não é para ser resolvido. É para provar a viabilidade do método" Eu testei alguns milhares de combinações usando força bruta, seu método é realmente seguro
> 
> Posso ter o link desse tweet, por favor? Eu não sabia que ele tuitou sobre isso nem achei nada quando li seus posts no Twitter.

https://x.com/coreylphillips/status/1373248775947481088

**Comment by glitchx7** [comment]
> > > Corey Phillips disse em seu twitter "Não é para ser resolvido. É para provar a viabilidade do método" Eu testei alguns milhares de combinações usando força bruta, seu método é realmente seguro
> > 
> > 
> > Posso ter o link desse tweet, por favor? Eu não sabia que ele tuitou sobre isso nem achei nada quando li seus posts no Twitter.
> 
> https://x.com/coreylphillips/status/1373248775947481088 It's in a comment

then why did he added if u managed to claim lmk, ngl i spent some time researching this

**Comment by lucasdmt** [comment]
> > > > Corey Phillips disse em seu twitter "Não é para ser resolvido. É para provar a viabilidade do método" Eu testei algumas milhares de combinações usando força bruta, seu método é realmente seguro
> > > 
> > > 
> > > Posso ter o link desse tweet, por favor? Eu não sabia que ele tuitou sobre isso nem achei nada quando li seus posts no Twitter.
> > 
> > 
> > https://x.com/coreylphillips/status/1373248775947481088 Está em um comentário
> 
> então por que ele adicionou se você conseguiu reivindicar lmk, ngl passei algum tempo pesquisando isso



> > > > Corey Phillips disse em seu twitter "Não é para ser resolvido. É para provar a viabilidade do método" Eu testei algumas milhares de combinações usando força bruta, seu método é realmente seguro
> > > 
> > > 
> > > Posso ter o link desse tweet, por favor? Eu não sabia que ele tuitou sobre isso nem achei nada quando li seus posts no Twitter.
> > 
> > 
> > https://x.com/coreylphillips/status/1373248775947481088 Está em um comentário
> 
> então por que ele adicionou se você conseguiu reivindicar lmk, ngl passou algum tempo pesquisando isso

brother, I still think the password is a related phrase, I just don't understand why he commented that, anyway we keep trying

**Comment by lucasdmt** [comment]
Interestingly, if you go back through some transactions on the blockchain you will find one with the op return 'congrats!'

**Comment by jmr2704** [comment]
Share the Blockchain link please

**Comment by lucasdmt** [comment]
use mempool to find

**Comment by SmartArt09** [comment]
> Share the Blockchain link please

The what? Did you mean the link to the address balance?
If so, you can use: https://www.blockchain.com/explorer/addresses/btc/bc1qcyrndzgy036f6ax370g8zyvlw86ulawgt0246r

**Comment by mosa2100** [comment]
Hey, you used the same method but set a password to prevent anyone from accessing the wallet, and it is impossible to find this password. This will reinforce the idea that your method is very secure and uncrackable. Curse you

**Comment by SmartArt09** [comment]
> Hey, you used the same method but set a password to prevent anyone from accessing the wallet, and it is impossible to find this password. This will reinforce the idea that your method is very secure and uncrackable. Curse you

Yeah, apparently that's what he intended with this second part of the whole puzzle - "show the viability of the method" and that "it is not meant to be solved"....

Also, it's not *his* method that is unsolvable, but rather a known fact that an additional password (the optional BIP39 passphrase) can add that much security to your wallet.

**Comment by mosa2100** [comment]
> (عبارة المرور الاختيارية BIP39)

Who said that? It might be different

**Comment by gustavocristovam** [comment]
> > Hey, you used the same method but set a password to prevent anyone from accessing the wallet, and it is impossible to find this password. This will reinforce the idea that your method is very secure and uncrackable. Curse you
> 
> Yeah, apparently that's what he intended with this second part of the whole puzzle - "show the viability of the method" and that "it is not meant to be solved"....
> 
> Also, it's not _his_ method that is unsolvable, but rather a known fact that an additional password (the optional BIP39 passphrase) can add that much security to your wallet.

Therefore, with a long BIP39 like this, it is almost impossible to discover, I believe the keyword is in the Image, or some hint in the 12M dollar transaction.

**Comment by SmartArt09** [comment]
> > > Hey, you used the same method but set a password to prevent anyone from accessing the wallet, and it is impossible to find this password. This will reinforce the idea that your method is very secure and uncrackable. Curse you
> > 
> > Yeah, apparently that's what he intended with this second part of the whole puzzle - "show the viability of the method" and that "it is not meant to be solved"....
> > 
> > Also, it's not _his_ method that is unsolvable, but rather a known fact that an additional password (the optional BIP39 passphrase) can add that much security to your wallet.
> 
> Therefore, with a long BIP39 like this, it is almost impossible to discover, I believe the keyword is in the Image, or some hint in the 12M dollar transaction.

The BIP39 is 24 words, which is one of the 2 most common lengths of a mnemonic phrase. 
Also, the optional passphrase may not necessarily be related to image or the transaction hash only (not that I tried all those ways), it can also be related to something personal (a lot of which I did try but none worked obviously).

**Comment by gustavocristovam** [comment]
Something personal, we'll never find out!

**Comment by SmartArt09** [comment]
> Something personal, we'll never find out!
> 

True.

**Comment by gustavocristovam** [comment]
Have you tried all possible types of steganography on images?

**Comment by SmartArt09** [comment]
> Have you tried all possible types of steganography on images?

I did try on the surface level, but could find anything (visually). I didn't try anything too deep, though.

**Comment by mosa2100** [comment]
> > Have you tried all possible types of steganography on images?
> 
> I did try on the surface level, but could find anything (visually). I didn't try anything too deep, though.
Don't bother yourself; everyone has given up on these trivial things. No one can solve it except the one who created it.

**Comment by mosa2100** [comment]
Don't bother yourself; everyone has given up on these trivial things. No one can solve it except the one who created it.
